### PR TITLE
use install script to install pytorch for docs

### PR DESCRIPTION
**Pull request by jpchen**
addresses #931

rtd cannot get new branches without a dev merge. so either we can merge this to test or merge another pr

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -189,10 +189,6 @@ def setup(app):
...
 
-# @jpchen's hack to get rtd builder to install pytorch
+# @jpchen's hack to get rtd builder to install latest pytorch
 if 'READTHEDOCS' in os.environ:
-    os.system('pip install awscli')
-    os.system('aws s3 --no-sign-request sync s3://pyro-ppl/ci tmp --exclude "*" --include "*-cp27-*";')
-    os.system('pip install tmp/*')
-    os.system('rm -r tmp')
-    os.system('pip install -e .')
+    os.system('sh scripts/install_pytorch.sh')
```
Comment (neerajprad) [inline docs/source/conf.py]: If this doesn't work, it could be because the installation script assumes bash and I'm not sure what `sh` points to here.

**Comment by neerajprad** [comment]
So should we merge this and give it a try?

**Comment by jpchen** [comment]
yeah I'll patch it if there's an issue. seems to be no other way

**Comment by jpchen** [comment]
```
sh: 0: Can't open scripts/install_pytorch.sh
```
might be a permissions issue. now that the branch exists ill test and follow up

**Pull request merged**